docker/Dockerfile

```diff
@@ -16,15 +16,18 @@ RUN set -ex \
  && useradd -u $USER_ID -r -g conda -d /home/conda -s /bin/bash -m conda \
  && chmod 755 /home/conda \
  && echo 'conda ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers \
- && curl -sSL https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -o miniconda.sh \
- && bash miniconda.sh -b -p /opt/conda \
- && rm miniconda.sh \
+ && curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh" \
+ && bash Miniforge3-$(uname)-$(uname -m).sh -b -p /opt/conda \
+ && rm Miniforge3-$(uname)-$(uname -m).sh \
  && touch /opt/conda/conda-meta/pinned \
  && export PATH=/opt/conda/bin:$PATH \
+ && conda config --prepend channels conda-forge \
+ && conda config --append channels nodefaults \
+ && conda config --set channel_priority strict \
  && conda config --set show_channel_urls True \
- && conda update --all --yes \
- && conda install --yes --file /tmp/requirements.txt \
- && conda install --yes --channel conda-forge --file /tmp/requirements_forge.txt \
+ && conda update --all --yes --channel conda-forge --override-channels \
+ && conda install --yes --channel conda-forge --override-channels --file /tmp/requirements.txt \
+ && conda install --channel conda-forge --override-channels --file /tmp/requirements_forge.txt \
```

Comment:
```suggestion
```

I don't think we need a different file for conda-forge requirements anymore. Would you mind removing this file (`requirements_forge.txt`) as well? Also, would you please add `seuptools` to `requirements.txt` to see if CI will succeed?

Reply:
There are two other lines in this file, 13 and 31 (before PR updates), that point to `requirements_forge.txt`.If we remove that file, we should remove these lines too.

Reply:
I've made these changes, but now I think the issue is with the CI configuration

See #57 - made a PR off of main with no functional code changes and the CI still failed with the same error seen here

Reply:
Thank you, @RKuttruff ! I see that you added pip install for setuptools. Hopefully, it will work!

Reply:
It starts the docker build now
I think I forgot some changes - I pinned some package versions in the requirements.txt

Reply:
Ok checks are passing